tests/misc/test_file_server.py

```diff
@@ -11,35 +12,27 @@
 #   The host must be configured with `website-https-only` set to true (which is the default config).
 
 def _header_equal(header, name, value):
-    regex = fr"{name}:\s?{value}"
+    regex = fr"{name}:\s?{re.escape(value)}"
     return re.match(regex, header) is not None
 
 def test_fileserver_redirect_https(host):
     path = "/path/to/dir/file.txt"
     ip = wrap_ip(host.hostname_or_ip)
-    process = subprocess.Popen(
-        ["curl", "-i", "http://" + ip + path],
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE
-    )
-    stdout, _ = process.communicate()
-    lines = stdout.decode().splitlines()
-    assert lines[0].strip() == "HTTP/1.1 301 Moved Permanently"
-    assert _header_equal(lines[2], "location", "https://" + ip + path)
+    res = commands.local_cmd(["curl", "-i", "http://" + ip + path])
+    lines = res.stdout.splitlines()
+    assert lines[5].strip() == "HTTP/1.1 301 Moved Permanently"
```

Comment:
Why line 5 now? What are the 5 new lines before it?

Reply:
```
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
```

Don't know where it comes from.

Reply:
An alias on your system?

Reply:
solved by using `-s` in curl and avoiding the extra lines.